src/ui/components/UrlExportSettings.tsx

```diff
@@ -195,6 +200,20 @@ export const UrlExportSettings = () => {
             />
           </Row>
         </>}
+
+      <Separator />
+      <Title size='xlarge' weight='bold'>About This Export</Title>
+      <h3>Message<Info width={200} label='Typically this will be a "commit message" for Git. Your organization may require a specific convention for these messages.' /></h3>
```

Comment:
Change to `<h3>Commit message`

This also fixes a bug where the info text is cut at the left side.

Reply:
I've changed it, but I actually purposefully did not use "Commit", because that is a term understood only by software developers and this plugin is Designer-facing. We actually chose your plugin over "Figma Tokens" because it allows non-technical designers to stay in normal "Figma Land". The URL export interface itself isn't even "Source-Control" specific (though I'm sure that's the most common usage.)
That's why I clarified in the `Info` tag that "_Typically_ this is a commit message..." but it doesn't necessarily have to be.

So, it is your call, but I'd consider a label that is more "Designer-First". Maybe "Message describing the changes"

If you're interested, our processing code is here on Github: unchartedsoftware/design-tokens